### ISSUE TITLE
Feature Request: ATB style Scenario generator for manual setup, repeatable scenarios and contract sharing

### ISSUE THREAD

**Issue by GraalOtonami** [MegaMek/mekhq#319]
Being able to manually edit scenarios and contract details to represent manual roles or player preference/story would be nice, though I can appreciate it not being high priority.
More desired to me are repeatable missions for things like Solaris Arenas, or the ability to save and send a contract to another player so that they can either back you up or even get contract pay for playing the Opfor.

**Comment by purpledragon2** [comment]
yes.  ability to edit scenarios and AtB contract details would be nice.  This is one of the things I think having multiple clients able to connect to the same server via web for MekHQ would be handy.
